config/routes.rb

```diff
@@ -4,5 +4,7 @@
   resources :dinosaurs do
     resources :reviews, only: [:create, :new, :destroy]
   end
-  resources :reviews, only: [:edit, :update, :destroy]
+  resources :reviews, only: [:edit, :update, :destroy] do
+    resources :comments, only: [:new, :update, :edit, :create, :destroy]
```

Comment:
Remove `edit`, `update`, & `destroy` until you write controller methods to handle these functions.